spec/controllers/pull_requests_controller_spec.rb

```diff
@@ -8,8 +8,16 @@
     let(:repo2) { create(:repository, organization: organization2) }
     let!(:pr1) { create(:pull_request, repository: repo1) }
     let!(:pr2) { create(:pull_request, repository: repo2) }
+    let!(:user1) { create(:github_user, login: "user_login") }
+    let!(:github_session) { instance_double("GithubSession") }
 
-    before { get :index, params: { organization_name: organization1.login } }
+    before do
+      allow(github_session).to receive(:token).and_return("token")
+      allow(github_session).to receive(:user).and_return(user1)
+      expect(subject).to receive(:authenticate_github_user).and_return(true)
```

Comment:
Do not use expect in before hook<br>Name your test subject if you need to reference it explicitly.<br>Prefer have_received for setting message expectations. Setup subject as a spy using allow or instance_spy.